packages/runtime/src/global.ts

```diff
@@ -28,8 +28,10 @@ export interface Federation {
 
 export const nativeGlobal: typeof global = (() => {
   try {
-    return new Function('return this')();
+    // get real window (incase of sandbox)
+    return document.defaultView;
   } catch {
+    // node env
     return globalThis;
```

Comment:
Global can be transformed by bundler. If we want root access to true global we may still want new function?